### ISSUE TITLE
Work Apps cant be put on homescreen

### ISSUE THREAD

**Issue by jessesmith4** [jooooscha/Olauncher#184]
If I long press to add apps to a home screen slot I can't add any "work" apps. I can see them in app drawer but just not add to home screen.  I am currently using shelter for the work apps and on graphene os. 

Environment information
- Olauncher CF Version: 1.2.6
- Install Source: neostore/fdroid
- Device: pixel 4a 5g
- Android: android 13 via Graphene os

**Comment by jooooscha** [comment]
What happens if you try to add them?

**Comment by jessesmith4** [comment]
Nothing.  If there was an app in the slot it clears it. So it looks like it would at first setup.  Just saying the word "app".

Also I noticed that if I have a version of the app in my non shelter account. I.e. contacts app it will actually let me place it on the home screen.  The only apps I have like that are system apps like contacts or browser.


________________________________

**Comment by jooooscha** [comment]
Weird, for me this is worling. Can anyone confirm this? Maybe its because of Android 13?

**Comment by jessesmith4** [comment]
You got it to work? Are you sure that the work app you added to the home screen wasn't also installed on the non work profile? If its installed on both it will work. 

I just tested it old essential phone with stock android 10 and the same version on the launcher and got the same behavior.  I also installed a work profile with a different app (from my actual work) and same thing. So I don't think its graphene or android 13 or shelter specifically that are causing the issue. I thought it worked in the past but now that I think about it the only work apps I put on the home screen were apps I had two copies of.
